### PR TITLE
Tmidi/issue#20

### PR DESCRIPTION
**Pull request by tmidi**
This is a quick fix for issue #20.

### CODE REVIEW

**Comment by nntoan** [comment]
Thanks @tmidi 
But I'm working on something similar to #17 and this will be included :+1:

File: aliases/general.aliases.sh [hunk 1]
```diff
@@ -27,7 +28,7 @@ alias mv='mv -iv'                           # Preferred 'mv' implementation
 alias mkdir='mkdir -pv'                     # Preferred 'mkdir' implementation
 alias ll='ls -lAFh'                         # Preferred 'ls' implementation
 alias less='less -FSRXc'                    # Preferred 'less' implementation
-alias nano='nano -W -$'                     # Preferred 'nano' implementation
+alias nano='nano -W'                 		# Preferred 'nano' implementation
```
Comment (Rathod-Sahaab) [inline aliases/general.aliases.sh]:
```suggestion
 alias nano='nano -W'                 		    # Preferred 'nano' implementation
```

Comment (Rathod-Sahaab) [reply aliases/general.aliases.sh]: To align with other **comments**

**Comment by nntoan** [comment]
Closed as it's a part of #39.

**Pull request closed without merging**